### DISCUSSION TITLE
Repository search, language tally seems incorrect

### DISCUSSION THREAD

**Discussion by kimryan** [community/community#21949] [New to GitHub]
<p>I am wondering how the repository totals by language are worked out? I havce several repositories written in Perl. I can search for them, but in the language totals table, Perl doesn’t appear.</p>
<p>For example, search for: street address  parse</p>
<p>My repsoitory appears on second page of results,  <a href="https://github.com/kimryan/Lingua-EN-AddressParse" rel="nofollow noopener">kimryan/Lingua-EN-<em>AddressParse</em></a></p>
<p>But language tally shows 1 repo for Elixir, Go and Haskell and none for Perl (when there is 1 repo)</p>
<p>My repo also has more stars then the Elixir etc ones</p>
<p>So the tally doesn’t seem accurate to me, unless I’m missing something.</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>It looks like it is showing only the first ten language results sorted first by number of results and then alphabetically by language name. Since there is only one Perl result and Elixir, Go, and Haskell sort alphabetically before Perl, it misses the cutoff. You’ll notice that Lua, R, and Typescript also aren’t in the list even though there is one repository that is marked as using each language in the list of results.</p>
